src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/model/Booking.kt

```diff
@@ -40,4 +40,9 @@ data class Booking(
     }
     return extractableSentences.toList()
   }
+
+  @JsonIgnore
+  val underEighteenAtEndOfCustodialPeriod: () -> Boolean = {
+    sentences.map { offender.getAgeOnDate(it.sentenceCalculation.releaseDate) < 18 }.all { it }
```

Comment:
Should this just be the latest release out of all the sentences? Also you don't need to map to a boolean then do all { it}. Just replace .map with .all

Reply:
D'oh! Of course!